gamma_bridge/storage.py

```diff
@@ -16,25 +56,50 @@ def __init__(self, enabled=False, endpoint=None, key=None, secret=None):
         self.endpoint = endpoint
         self.key = key
         self.secret = secret
+        self.response_data = None
+        self.response_data_content = None
+        super(GammaStorage, self).__init__()
```

Comment:
Please move it to the top of the `__init__` method.

Reply:
Done